examples/load_ui_base_instance.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import os
+
+os.environ["QT_PREFERRED_BINDING"] = "PySide"
```

Comment:
I get a weird thing in Python 2.7 if I don't specify this. Don't really understand why. I can remove it and you'll see in my next commit.